plugins/woocommerce-admin/client/settings/settings-slots.js

```diff
@@ -18,6 +18,10 @@ export const possiblyRenderSettingsSlots = () => {
 		},
 		{ id: 'wc_tax_settings_slotfill', scope: 'woocommerce-tax-settings' },
 		{ id: 'wc_settings_slotfill', scope: 'woocommerce-settings' },
+		{
+			id: 'wc_settings_general_site_visibility_slotfill',
+			scope: 'woocommerce-site-visibility-settings',
```

Comment:
I think we should use the same scope `woocommerce-settings` here. The reason this exists is so that we're not loading all of the app's Fills, just the ones that match the current scope. `woocommerce-settings` is already on the page so may as well use it

Reply:
Sounds good @psealock 👍 Updated in [Use woocommerce-settings scope](https://github.com/woocommerce/woocommerce/pull/45402/commits/01ff813ffdd6e26a7eeee96c5df567d42098ce01)